_i18n/fr.yml

```
@@ -0,0 +1,62 @@
+# 1.0.0
+langs:
+  fr: Français
+
+global:
+  404: Page non trouvée
+home:
+  title: |
+    Le pouvoir des <span class="red">travailleurs</span> dans l'industrie de la technologie
+  summary: Guidé par notre vision d'une industrie technologique inclusive et équitable, TWC s'organise pour renforcer le pouvoir des travailleurs grâce à l'auto-organisation et à l'éducation.
+  black_lives_matter:
+    title: Black Lives Matter
+    description: Nous encourageons les travailleurs technologiques à soutenir le mouvement Black Lives Matter dans leur lutte pour la justice. Dans ce sens, nous avons <a class="hover:white" href="/security">rassemblé ici une liste de guides de sécurité</a> que vous pouvez utiliser pour vous protéger et protéger vos camarades.
+  who_we_are:
+    title: Qui nous sommes
+    description: Nous sommes une coalition de travailleurs dans et autour de l'industrie technologique, des organisateurs syndicaux, des organisateurs communautaires et des amis.
+  why_we_organize:
+    title: Pourquoi nous organisons
+    description: Nous nous organisons pour l'activisme, l'engagement civique et l'éducation dans la Bay Area, Seattle, aux États-Unis et dans le monde.
+  who_we_support:
+    title: Qui nous soutenons
+    description: Nous travaillons en solidarité avec les mouvements existants pour la justice sociale, les droits des travailleurs et l'inclusion économique.
+  how_we_work:
+    title: Comment nous travaillons
+    description: |
+      Nous sommes une organisation démocratiquement structurée, entièrement composée de bénévoles et dirigée par des travailleurs. À ce stade, l'adhésion consiste à assister à des réunions en personne et à travailler sur les différents projets qui intéressent les gens. Nous organisons en ligne, mais l'IRL est au cœur de ce que nous faisons. Découvrez notre <a href="/community-guide">Guide communautaire</a> pour plus d'informations. Rejoignez <a href="/subscribe">Slack</a> pour entrer en contact avec un local de votre région.
```

```suggestion
      Nous sommes une organisation démocratiquement structurée, entièrement composée de bénévoles et dirigée par des travailleurs. À ce stade, l'adhésion consiste à assister à des réunions en personne et à travailler sur les différents projets qui intéressent les gens. Nous organisons en ligne, mais l'IRL est au cœur de ce que nous faisons. Découvrez notre <a href="/fr/community-guide">Guide communautaire</a> pour plus d'informations. Rejoignez <a href="/subscribe">Slack</a> pour entrer en contact avec un local de votre région.
```

This is where the French guide should be used (added `/fr/` prefix to guide url